README.md

```diff
@@ -243,50 +243,78 @@ Die Angabe *gemeinsamerHaushalt* ist nur beim zweiten Antragsteller relevant.
         "rentner": Rentner
     }
 
-Die zu befüllende Felder zur Beschäftigung ist abhängig von der ausgewählten Beschäftigungsart.<BR>Ist keine Beschäftigungsart gesetzt bzw. andere Felder, die nicht zur Beschäftigungsart passen,
-befüllt, werden sie ignoriert. Beispiel: *beschaeftigungsart=ARBEITER*, dann wird der Knoten *arbeiter* berücksichtigt
+Die `Beschaeftigungsart` bestimmt die Beschäftigung und damit das dazu korrespondierende Feld. Beispielsweise wird für die `beschaeftigungsart=ARBEITER`
+die Daten unter dem Knoten `arbeiter` genutzt, bei der `beschaeftigungsart=BEAMTER` entsprechend der Knoten `beamter`. Werden darüber hinaus weitere Felder befüllt so werden diese ignoriert.<BR/>
```

Comment:
Bitte kein HTML wenn es nicht nötig ist, wenn du einen Zeilenumbruch haben willst, kannst du in Markdown einfach eine Leerzeile einfügen.

```suggestion
die Daten unter dem Knoten `arbeiter` genutzt, bei der `beschaeftigungsart=BEAMTER` entsprechend der Knoten `beamter`. Werden darüber hinaus weitere Felder befüllt so werden diese ignoriert.

```

Reply:
Also für mich ist am Ende nur wichtig: Ich hätte es gerne gleich an beiden Stellen (Update und Import) einfach weils einheitlicher ist und somit besser verständlich. Wenn das jetzt doch auf Befindlichkeiten stößt, dann schließe ich erstmal den PR und wir können das gerne im Team noch mal besprechen und dann entsprechend anpassen. So wie es jetzt ist finde ich es zumindest nicht optimal.